### PR TITLE
added sqlite3 integration for managing client data using database tools

### PR DESCRIPTION
**Pull request by hitanshup**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -77,3 +100,29 @@ app.post('/add', function(req, res) {
...
+            db.run("CREATE TABLE if not exists user_info (shrunk TEXT UNIQUE, url TEXT, expiry TEXT)");
+        });
+    }
+}
+
+//adds new urls to the database
+function addUrlToDb(shrunk, url, expiry) {
+    db.serialize(function() {
+        var stmt = db.prepare("INSERT INTO user_info VALUES (?,?,?)");
+        stmt.run(shrunk, url, expiry);
```
Comment (Cisplatin) [inline app.js:119]: Does this sanitize data, or is this vulnerable to an SQL injection?

Comment (hitanshup) [reply app.js:119]: https://github.com/mapbox/node-sqlite3/issues/57 so I think yes

Comment (hitanshup) [reply app.js:119]: and by yes I mean yes it sanitizes :p lol

**Comment by Cisplatin** [comment]
+1

**Pull request merged**